library/src/main/java/com/novoda/downloadmanager/VersionOneToVersionTwoMigrator.java

```diff
@@ -0,0 +1,143 @@
+package com.novoda.downloadmanager;
+
+import android.util.Log;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.List;
+
+import static com.novoda.downloadmanager.DownloadBatchStatus.Status;
+
+class VersionOneToVersionTwoMigrator implements Migrator {
+
+    private static final int BUFFER_SIZE = 8 * 512;
```

Comment:
Maybe we could explain the size choice if there's some reasoning. Multiples of 512... disk blocks?

If we can express that in a short enough name that would be great, if not a comment would be better than nothing I think.

Reply:
There's no real reason we're going with this buffer size other than "we tried a few a different sizes and this one seemed to work". Any suggestions welcome here!

Reply:
Ah. Then even that I guess, and maybe not multiples of 512. It just lead me to believe it might be super important.